aws/resource_aws_codestarconnections_connection.go

```diff
@@ -43,6 +44,13 @@ func resourceAwsCodeStarConnectionsConnection() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validation.StringInSlice(codestarconnections.ProviderType_Values(), false),
 			},
+
+			"tags": {
+				Type:     schema.TypeMap,
+				Optional: true,
+				ForceNew: true,
```

Comment:
Updating resource tags should not require resource recreation -- we can add an `Update` function that only handles `tags`. 👍